WordPress/Classes/Services/ReaderSiteService.m

```diff
@@ -83,6 +83,9 @@ - (void)followSiteWithID:(NSUInteger)siteID success:(void(^)())success failure:(
             return;
         }
         [service followSiteWithID:siteID success:^(){
+            if (success) {
+                success();
+            }
             [WPAnalytics track:WPAnalyticsStatReaderSiteFollowed];
```

Comment:
I have to ask the dumb question here...should all of the `WPAnalytics` calls be in the `if (success) ` block?